Cubical/Foundations/Prelude.agda

```diff
@@ -301,6 +306,16 @@ funExt⁻ : {B : A → I → Type ℓ'}
   → ((x : A) → PathP (B x) (f x) (g x))
 funExt⁻ eq x i = eq i x
 
+_≡$_ = funExt⁻
```

Comment:
How often does one want an infix version of this? Not sure how useful it is, but I'm not strongly opposed to it :-)

Reply:
What `funExt⁻` really does is applying a path between functions to an element of the functions' domain, so it is really an application operator. In the HoTTbook it is called `happly`, in the agda standard library it is called `cong-app`. In my experience, one almost always wants an infix version of this, and especially when happlying to multiple arguments.

Reply:
Ok, sounds good to me!